react/SearchProductsProgressBar.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { path } from 'ramda'
+import { useSearchPage } from 'vtex.search-page-context/SearchPageContext'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = ['progressBarContainer', 'progressBar', 'progressBarFiller']
+
+const SearchProductsProgressBar = () => {
```

Comment:
I think you can rename this for `ProductsProgressBar`. This component is in the `search-result` app, so there is no need to use "Search"  as a prefix. This change also makes the naming more compatible with `ProductCountPerPage`